app/src/main/res/values/strings.xml

```diff
@@ -440,4 +440,9 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_housenumber_conscription_number_required">A conscription number must be specified</string>
     <string name="quest_cycleway_answer_contraflow_cycleway">Also cycleway on other side</string>
     <string name="need_specify_both_sides">You need to specify both sides</string>
+    <string name="quest_internet_access_name_title">What kind of connection does "%s" offer?</string>
+    <string name="quest_internet_access_wlan">WLAN</string>
+    <string name="quest_internet_access_wired">Wired</string>
+    <string name="quest_internet_access_terminal">Terminal</string>
```

Comment:
Not sure if "Wired" and "Terminal" would be understood clearly by anyone.

Reply:
Maybe "Wired" can be understood more clearly if it's "Wired (LAN)"...
But for "Terminal" I have no idea how it can be more clearly...

Reply:
"There is a computer with internet access"?